lib/rubocop/cop/performance/hash_each_methods.rb

```diff
@@ -55,17 +67,32 @@ def autocorrect(node)
           caller, first_method = *receiver
 
           lambda do |corrector|
-            if receiver.method?(:hash)
-              correct_hash(node, corrector)
-            else
+            if first_method.eql?(:keys) || first_method.eql?(:values)
```

Comment:
Not sure why hash equality (`#eql?`) is used to compare symbols? 🙂 

```ruby
%i[keys values].include?(first_method)
```

or

```ruby
first_method == :keys || first_method == :values
```

Same applies to all uses of `#eql?`.